CHANGELOG.md

```diff
@@ -74,6 +76,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Bindings for Super/Command + F1-F12
 - Automatic config generation
+- Deprecated `scrolling.faux_multiplier`, use `scrolling.multiplier` for both usual and faux
+scrolling. You can enable/disable faux scrolling using `\e[?1007h`/`\e[?1007l` escape sequences
+respectively.
```

Comment:
```
- Deprecated `scrolling.faux_multiplier`, the alternate scroll escape can now be used to disable it and `scrolling.multiplier` controls the number of scrolled lines
```

This should be a little shorter and reduce repetition with the other entry. If a change log entry spans multiple lines, it should also be indented by two additional columns like this:

```
- First line
    Second line
```